### DISCUSSION TITLE
Welcome to FRET Discussions!

### DISCUSSION THREAD

**Discussion by anmavrid** [NASA-SW-VnV/fret#28] [Announcements]
## Welcome to FRET discussions

This is a space for connecting with the FRET community and fostering open communication. We encourage you to use it to:

* Ask questions about FRET.

* Discuss requirement examples and case studies.

* Share ideas, feedback, or suggestions.

We’re excited to hear from you!

— The FRET Team

**Comment by kaci2002** [comment]
Hello, the GUI of FRET on my centos become a white screen after I click the REALIZABILITY and nothing is outputed on the  terminal. Could you tell me what happen? Thanks.

**Reply by andreaskatis** [reply]
Hi @kaci2002,

I have created an issue related to your problems in our Issues tab: https://github.com/NASA-SW-VnV/fret/issues/48 

Please use that thread to communicate any additional information.

Best Regards,

Andreas
